### ISSUE TITLE
Better sizing for default and minimum column widths

### ISSUE THREAD

**Issue by gtackett** [NationalSecurityAgency/ghidra#2989]
**Is your feature request related to a problem? Please describe.**
Not related to a problem as such--more of an inconvenience

**Describe the solution you'd like**
Default column widths in some tables are really too wide. The minimum widths for the same columns is also needlessly large.

The particular window where I currently observe this is the **Fid DB Search Results** window. Many of the columns are overly wide. For example, the 'size' column (the name may be slightly different) looks wide enough to accommodate 16 digits. The default and minimum sizes could both be at least a few digits narrower without inconveniencing anyone. (In any case, Ghidra as we know it today would certainly  choke before a function got as large as 10^16 code units. Perhaps by the time Ghidra version 10^2 is on the horizon? 😆 )

**Describe alternatives you've considered**

1. Persuade the authorities here to purchase a 40 in. monitor.
2. Live with the inconvenience.

**Comment by dragonmacher** [comment]
This is likely specific to the FID table you mentioned.   Many (most?) tables specify reasonable default values for columns.

**Comment by gtackett** [comment]
You may be correct, @dragonmacher . I think some of my annoyance with column widths may stem from other tables where some columns may only need a few characters for the data, but have rather wide headings, e.g. "Number of References" (not sure if if that's exactly the wording). 

In such cases the wide headings cause the default column width to be a lot wider than is needed for the data. Of course one can shrink the column width (with possibly a few exceptions beyond the **Fid DB Search Results** table I originally mentioned above), at the expense of having the column header "ellipsis-ified", which makes "Number of _x_" hard to distinguish from "Number of _y_".

**Comment by dragonmacher** [comment]
> "Number of x" hard to distinguish from "Number of y".

True.  In this case the column should have a better name, such as `Y Count` or `#`, with a tooltip that provides a description.
